src/widgets/Tabs/PatientSelect.js

```diff
@@ -139,9 +139,17 @@ const PatientSelectComponent = ({
   const [isQrModalOpen, toggleQrModal] = useToggle();
 
   const onQrCodeRead = ({ data }) => {
-    // TODO: Some validation might be good here but don't know format..
+    // TODO: Some validation/data sanitisation might be required here but don't know format yet...
     toggleQrModal();
-    ToastAndroid.show(`${data} scanned`, ToastAndroid.LONG);
+
```

Comment:
It would be awesome to have some haptic feedback if that's possible - it works for me with the covid tracer app anyway :)
Just something to let the user know that the scan happened and they can put the tablet down now

Reply:
Ooh this is a neat idea, will have a looksies 👀

Reply:
Stole the package name from the covid tracer app and added some feedback on successful scan 🤣  ed3d4123